deployment/helm/eclipse-ditto/files/nginx/nginx.conf

```diff
@@ -81,7 +81,8 @@ http {
     location /status {
       include nginx-cors.conf;
 
-      proxy_pass                    http://gateway:8080/overall/status;
+      # change proxy_pass to http://gateway:8080/overall/status for newer versions;
+      proxy_pass                    http://gateway:8080/status;
```

Comment:
change this to `proxy_pass                    http://gateway:8080/overall/status;`